study-builder/studies/pancan/snippets/question-diagnosed-primary-cancer.conf

```diff
@@ -0,0 +1,2516 @@
+{
+  include required("../../snippets/picklist-question-single-autocomplete.conf"),
+  "stableId": "DIAGNOSED_PRIMARY_CANCER",
+  "promptTemplate": {
+    "templateType": "HTML",
+    "templateText": "$prequal_diagnosed_primary_cancer_prompt"
+    "variables": [
+      {
+        "name": "prequal_diagnosed_primary_cancer_prompt",
+        "translations": [
+          { "language": "en", "text":${i18n.en.prequal_diagnosed_primary_cancer_prompt} },
+          { "language": "es", "text": ${i18n.es.prequal_diagnosed_primary_cancer_prompt} }
+        ]
+      }
+    ]
+  },
+  "picklistOptions": [
```

Comment:
This is good for now, but let's make a note to come back and group these into nested options.